inference-engine/src/vpu/graph_transformer/src/stages/loop_end.cpp

```diff
@@ -14,6 +14,8 @@ namespace vpu {
 
 namespace {
 
+const int32_t dynamicIterationNum = -1;
```

Comment:
Is it possible to reuse the same `dynamicIterationNum` declaration for both `LoopStart` and `LoopEnd`?